staff/demi/tasklist/data/taskData.js

```diff
@@ -0,0 +1,74 @@
+var _ = require('lodash');
+
+let tasks = []
+const ids = []
+
+ taskData = {
+    
+    list() { return tasks },
+
+    retrieve(task) {
+
+        function isTask(_task) { 
+            var _task = _task.id.toString();
+            return _task === task;
+        }
+
+        var task = tasks.find(isTask)
+        
+        if (task) return task 
+
+        throw Error('Task does not exist.')
+    },
+
+    create(task, status) {
+        
+        var id = _.random(1000, 9000);
+        ids.push(id)
+
+        tasks.push({ task, status, id })
+
+    }, 
+
+    delete() {
+
+        if (tasks.length === 0) throw Error('There are no tasks to delete')
+
+        tasks.splice(0,tasks.length)
+
+    },
+
+    deleteSelected(id, task) {
```

Comment:
It should only one method --> **delete(id)**
If you want delete all task, you should do it in taskLogic